blockstack/lib/nameset/namedb.py

```diff
@@ -501,7 +517,7 @@ def log_commit( self, block_id, vtxindex, op, opcode, op_data ):
         return
 
 
-    def log_reject( self, block_id, vtxindex, op, op_data ):
+    def log_reject( self, block_id, vtxindex, op, op_data, do_print=True):
```

Comment:
This would be much more useful with an optional "reason" field -- we could add the reason just to tokentransfers for now -- that way, you'd know if the transfer operation was rejected because the address doesn't have enough balance, the consensus hash is invalid, or the sender == recipient

Reply:
Fixed.  Added this to `tokentransfer.py` only for now.